Cargo.toml

```diff
@@ -20,24 +20,25 @@ doctest = false # and no doc tests
 [dependencies]
 getrandom = { version = "0.2", features = ["std"] }
 env_logger = "0.9"
-libffi = "3.0.0"
-libloading = "0.7"
 log = "0.4"
 shell-escape = "0.1.4"
 rand = "0.8"
 smallvec = "1.7"
 
+
 # A noop dependency that changes in the Rust repository, it's a bit of a hack.
 # See the `src/tools/rustc-workspace-hack/README.md` file in `rust-lang/rust`
 # for more information.
 rustc-workspace-hack = "1.0.0"
 measureme = "10.0.0"
 
-# Enable some feature flags that dev-dependencies need but dependencies
-# do not.  This makes `./miri install` after `./miri build` faster.
 [target."cfg(unix)".dependencies]
 libc = "0.2"
+libffi = "3.0.0"
+libloading = "0.7"
 
+# Enable some feature flags that dev-dependencies need but dependencies
+# do not.  This makes `./miri install` after `./miri build` faster.
```

Comment:
I think this comment is outdated, it refers to some feature flags we added on regular dependencies that were only needed for some dev-deps.